### ISSUE TITLE
[Grid] Screen reader is not announcing "No Data Found" when filtering returns no results

### ISSUE THREAD

**Issue by msftedad** [telerik/kendo-angular#4629]
### Description

Screen reader is not announcing the search result information on giving invalid data in the search edit field in windows.
The absence of screen reader announcements for invalid search results affects users with visual impairments, including blind and low-vision individuals, as well as users with cognitive disabilities who depend on auditory feedback to understand and correct errors, limiting their ability to use the search functionality independently.

### Steps To Reproduce

1. Open the URL: [Agora Home Page (PPE)](https://agoradev-ppe.azurewebsites.net/home?dashboard-id=RnRToolsHomePage) in latest edge browser.
2. Press tab key to move to the 'Save as' control present in the page and activate it using enter key.
3. Save as dialog gets displayed.
4. Press tab key to move to the 'workspace to be saved as in' edit field and activate it using 'Alt + down arrow key'.
5. Enter any invalid data in the search edit field.
6. Now Verify whether screen reader is announcing the search result information on giving invalid data in the search edit field in windows or not.

### Screenshots or video

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/a45ec9e1-2ba7-40d5-8c2b-ba9e2e1ebe51" />

https://github.com/user-attachments/assets/75f3b41a-caad-47f4-81a2-9e053d62db62

### Actual Behavior

Screen reader is not announcing the search result information on giving invalid data in the search edit field in windows. Screen reader is remaining silent after giving invalid data in the edit field.
Observation: Screen reader is narrating the displayed result information on giving valid data in the edit field.

### Expected Behavior

Screen reader should announce the displayed search result information on giving invalid data in the search edit field in windows. Screen reader should narrate as "No data found'.

### Browser

Chrome, Microsoft Edge

### Browser version

latest

### OS type

MacOS, Windows

### OS version

Windows 11 Enterprise Insider Preview Version 24H2(OS Build 26100.2892)

**Comment by msftedad** [comment]
Hi Team, Any update on this issue?
